docs/instrumentation-list.yaml

```diff
@@ -1213,11 +1213,6 @@ opensearch:
     target_versions:
       javaagent:
       - org.opensearch.client:opensearch-rest-client:[1.0,)
-  - name: opensearch-java-2.0
-    srcPath: instrumentation/opensearch/opensearch-java-2.0
-    scope:
-      name: io.opentelemetry.opensearch-java-2.0
-    target_versions: {}
```

Comment:
this was just from me experimenting with writing new instrumentation and it got committed by mistake